resources/views/auth/login.blade.php

```diff
@@ -22,29 +22,37 @@
                 <div class="box-header">
                     <h3 class="box-title">Sign in</h3>
                 </div>
-                <form method="POST" action="/auth/login">
+                    {!! Form::open(['action'=>'Auth\AuthController@postLogin']) !!}
                     <div class="box-body">
-                        {!! csrf_field() !!}
+                        @if (count($errors) > 0)
+                            @foreach ($errors->all() as $error)
+                                <div class="alert alert-danger alert-dismissable">
+                                    <button type="button" class="close" data-dismiss="alert"
+                                            aria-hidden="true">&times;</button>
+                                    <i class="icon fa fa-ban"></i>{{ $error }}
+                                </div>
+                        @endforeach
+                    @endif
```

Comment:
Indentation wrong in two lines.

Only these changes are correct for issue #153. Other changes are coming due to merge on other issue. If we merge this PR, all other PR will be merged automatically. Thus, before we can merge this PE, we need to merge other PRs and update this branch from master.